src/hooks.ts

```diff
@@ -162,3 +170,48 @@ export function usePointPosition(config: UsePositionConfig[]) {
     )
     .join(' ');
 }
+interface UseShiftOptions extends DualAxisOptions {
+  xShift?: number | string;
+  yShift?: number | string;
+}
+export function useShift(options: UseShiftOptions) {
+  const { axisContext, plotWidth, plotHeight } = usePlotContext();
+  const {
+    horizontalAxisId = 'x',
+    verticalAxisId = 'y',
```

Comment:
Those should be mandatory probably to avoid bugs

The default values are already set by the components which use this hook

Reply:
problem fixed thank you

Reply:
maybe we should discuss about this because it's used in other hooks

Reply:
Public (exported in the main index file) hooks and components should definitely have this optional so that the usual default case is handled with as few props and options as possible

But does it make sense to make useShift and useShiftInverted public?

If not then we should have those properties mandatory and make them private internal functions

If yes then what is the use case? It seems to me like the only purpose of those hooks is to be used internally but I might be missing something

Reply:
yes useShift is an internal private function
I totaly agree with you
Thanks